src/org/team1540/kingbass/OI.java

```diff
@@ -107,7 +107,7 @@
 
     // driverDPadLeft.whenPressed(new AutoShift());
 
-    driverLeftStick.whenPressed(new DriveToObject());
+    driverRightTrigger.whenPressed(new ReverseDriveDirection());
```

Comment:
Should DriveToObject be bound to something? Just because it's untested doesn't really mean it shouldn't be bound